fuse/src/main/java/tachyon/fuse/TachyonFuseFs.java

```diff
@@ -65,7 +64,7 @@
   private static final long[] UID_AND_GID = TachyonFuseUtils.getUidAndGid();
 
   private final TachyonConf mTachyonConf;
-  private final TachyonFileSystem mTFS;
+  private final FileSystem mTFS;
```

Comment:
Since `tfs` -> `fs` or `fileSystem` will be required in more places, do you mind if I include those changes in that PR?